### DISCUSSION TITLE
有的图片加载的出来有的加载不出来，好奇怪？

### DISCUSSION THREAD

**Discussion by Dxin02** [YunYouJun/hexo-theme-starry#312] [Q&A 疑难杂症]
<img width="596" alt="2" src="https://user-images.githubusercontent.com/87813631/172165832-207d1251-206a-4c22-8678-6de38fe9bab7.png">

<img width="526" alt="1" src="https://user-images.githubusercontent.com/87813631/172165805-dd6baf31-523e-4c52-ae4b-fdab4cbdd955.png">

有的图片加载的出来有的加载不出来，好奇怪？

**Comment by Dxin02** [comment]
是图片透明度变成0了，不是加载不出来。

**Reply by YunYouJun** [reply]
能否提供可复现地址。

**Reply by Dxin02** [reply]
纯小白，不太懂。不过本地打开md文件显示是没有问题的。就像这样子，有的图片变透明看不见了，图片都是放在一个oss的，代码也差不多。
![image](https://user-images.githubusercontent.com/87813631/172181790-4f0c70ac-1679-4d5b-ab02-65ce773349b1.png)
![image](https://user-images.githubusercontent.com/87813631/172181346-63487758-194f-4d23-a693-d786233e2883.png)

**Reply by YunYouJun** [reply]
版本号？
在线预览地址？

**Reply by Dxin02** [reply]
hexo 是6.2.0 ，主题应该是1.8.x
在线的话是这个地址：https://ohhhhhhhhhhhhhhhhhhhhh.github.io/

**Reply by YunYouJun** [reply]
试试升级到 1.10.x ？

**Reply by Dxin02** [reply]
破案了，是写markdown的时候给图片设置的style=“zoom：xx%”可能跟什么冲突了，显示不了的那些图片都是出现一下再变透明然后就不复原了。不过只是部分图片会出现问题也是很奇怪（）
主题是最新的1.10.x，上边说错了。
以后就不用zoom来缩放图片了，问题解决（bushi
谢谢你，辛苦了！

**Reply by YunYouJun** [reply]
可以通过 width 百分比，或 `transform: scale(xx)` 等方式进行缩放。

https://developer.mozilla.org/en-US/docs/Web/CSS/zoom
